app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1435,6 +1444,14 @@ DeviceDelegateOpenXR::EnterVR(const crow::BrowserEGLContext& aEGLContext) {
   XrSessionCreateInfo createInfo{XR_TYPE_SESSION_CREATE_INFO};
   createInfo.next = reinterpret_cast<const XrBaseInStructure*>(&m.graphicsBinding);
   createInfo.systemId = m.system;
+
+  if (OpenXRExtensions::IsExtensionSupported(XR_EXTX_OVERLAY_EXTENSION_NAME)) {
+    XrSessionCreateInfoOverlayEXTX overlayInfo { .type = XR_TYPE_SESSION_CREATE_INFO_OVERLAY_EXTX, .createFlags = 0, .sessionLayersPlacement = 0 };
```

Comment:
Nit: I would break this long line into several lines, one for each member. It helps read-ability IMO.